src/modules/roc_pipeline/config.h

```diff
@@ -168,8 +165,8 @@ struct ReceiverSessionConfig {
     //! Target latency, nanoseconds.
     core::nanoseconds_t target_latency;
 
-    //! Channel mask.
-    packet::channel_mask_t channels;
+    //! Sample spec.
+    audio::SampleSpec sample_spec;
```

Comment:
Could you please take a look at this comment? https://github.com/roc-streaming/roc-toolkit/pull/430#discussion_r536713364

It suggest to get rid of ReceiverSessionConfig::sample_spec at all and only use sample_spec from format and common_config. What do you think, can we do it?